pom.client.xml

```diff
@@ -558,6 +558,22 @@
                 <title>Azure Core - Management</title>
                 <packages>com.azure.core.management*</packages>
               </group>
+              <group>
+                <title>Azure Core - Serializer - Apache Avro</title>
```

Comment:
I would have this worded slightly differently perhaps - 'Azure Core - Serializer - Avro - Apache', etc.

Reply:
Updated to use this convention